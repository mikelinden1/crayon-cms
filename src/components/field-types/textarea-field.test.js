import TextAreaField from './textarea-field';

const props = {
    name: 'test',
    value: 'value',
    onChange: () => {},
    disabled: false
};

it('TextAreaField matches snapshot', () => {
    const wrapper = shallow(<TextAreaField {...props} />);
    expect(wrapper).toMatchSnapshot();
});
