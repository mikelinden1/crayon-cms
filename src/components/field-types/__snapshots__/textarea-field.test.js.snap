// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextAreaField matches snapshot 1`] = `
<textarea
  className="form-control"
  disabled={false}
  name="test"
  onChange={[Function]}
  rows={4}
  value="value"
/>
`;
